test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -53,5 +54,39 @@ public void TextFeaturizerWorkout()
             CheckEquality("Text", "featurized.tsv");
             Done();
         }
+
+        [Fact]
+        public void TextTokenizationWorkout()
+        {
+            string sentimentDataPath = GetDataPath("wikipedia-detox-250-line-data.tsv");
+            var data = TextLoader.CreateReader(Env, ctx => (
+                    label: ctx.LoadBool(0),
+                    text: ctx.LoadText(1)), hasHeader: true)
+                .Read(new MultiFileSource(sentimentDataPath));
+
+            var invalidData = TextLoader.CreateReader(Env, ctx => (
+                    label: ctx.LoadBool(0),
+                    text: ctx.LoadFloat(1)), hasHeader: true)
+                .Read(new MultiFileSource(sentimentDataPath));
+
+            var est = new WordTokenizer(Env, "text", "words")
```

Comment:
>WordTokenizer [](start = 26, length = 13)

Whats the purpose of TransformWrapper? Is it to quickly manufacture transformers from existing transform? because I don't see the Pigsty extensions to it? Will it require additional work?

Reply:
Yes, this is exactly for that.

Pigsty comes extra, see the subsequent iterations

---
In reply to: [218245812](https://github.com/dotnet/machinelearning/pull/931#discussion_r218245812) [](ancestors = 218245812)